Ports/chocolate-doom/patches/0002-Add-serenity-support-to-config.sub.patch

```diff
@@ -0,0 +1,21 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Mike Swanson <mikeonthecomputer@gmail.com>
+Date: Wed, 10 May 2023 18:49:46 -0700
+Subject: [PATCH] Add serenity support to config.sub
+
+---
+ autotools/config.sub | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/autotools/config.sub b/autotools/config.sub
+index 1d8e98bcee23a0421e4fafe9a6c9ac75180cff25..d294bcb86a41039d272b72d14c7546cd782c9f12 100755
+--- a/autotools/config.sub
++++ b/autotools/config.sub
+@@ -1364,6 +1364,7 @@ case $os in
+ 	# Each alternative MUST end in a * to match a version number.
+ 	# -sysv* is not here because it comes later, after sysvr4.
+ 	-gnu* | -bsd* | -mach* | -minix* | -genix* | -ultrix* | -irix* \
++	      | -serenity* \
+ 	      | -*vms* | -sco* | -esix* | -isc* | -aix* | -cnk* | -sunos | -sunos[34]*\
+ 	      | -hpux* | -unos* | -osf* | -luna* | -dgux* | -auroraux* | -solaris* \
+ 	      | -sym* | -kopensolaris* | -plan9* \
```

Comment:
Instead of manually patching this, have you tried setting `use_fresh_config_sub='true'`?